conf/main/logback.xml

```diff
@@ -21,32 +21,46 @@
     <property name="graknLogFileName" value="grakn.log"/>
     <property name="graknPPLogFileName" value="grakn-postprocessing.log"/>
 
+    <!--Configure a properties file (located on the classpath) from which values can be referenced-->
     <property resource="grakn.properties"/>
 
+    <!--Extract the directory of the grakn logs from the properties file. If the path is relative it
+    will be appended to the directory containing grakn (system property grakn.dir). Otherwise the absolute
+    path is used. -->
+    <if condition='property("log.dirs").startsWith("/")'>
```

Comment:
No, but that does not mean I won't judge you for putting conditional XML statements in